packages/account/src/Sections/Profile/PersonalDetails/input-group.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type TInputGroup = {
+    children: React.ReactNode;
+    className: string;
+};
+
+const InputGroup = ({ children, className }: TInputGroup) => (
```

Comment:
cant we use React.PropsWithChildren instead?
type TInputGroup = {
    className: string;
};

const InputGroup = ({ children, className }: React.PropsWithChildren<TInputGroup>) => (